packages/web-twig/src/Resources/components/TextField/stories/TextFieldV1DisabledFeature.twig

```diff
@@ -1,28 +1,15 @@
-<div class="spirit-v1-box-field-disabled docs-FormFieldGrid">
-    <h2 class="docs-Heading">V1 Feature &mdash; new disabled state</h2>
-
-    <TextField
-        hasPasswordToggle
-        id="textfieldPasswordToggleDisabledFeature"
-        isDisabled
-        label="Password Toggle"
-        name="textfieldPasswordToggleDisabledFeature"
-        value="Disabled"
-    />
-    <TextField
-        id="textfieldDisabledFeature"
-        isDisabled
-        label="Label"
-        message="Message"
-        name="textfieldDisabledFeature"
-        placeholder="Placeholder"
-    />
-    <TextField
-        id="textfieldDisabledFilledFeature"
-        isDisabled
-        label="Label"
-        name="textfieldDisabledFilledFeature"
-        placeholder="Placeholder"
-        value="Filled"
-    />
-</div>
+<TextField
+    id="textfieldDisabledV1"
+    isDisabled
+    label="Label"
+    name="textfieldDisabledV1"
+    placeholder="Placeholder"
+/>
+<TextField
```

Comment:
This one is marked as required in the web package:

![obrazek](https://github.com/lmc-eu/spirit-design-system/assets/5614085/9d5e4484-8462-4a19-aae9-325801dc4893)